README.md

```diff
@@ -1,7 +1,9 @@
 Micro JS Enum
 =============
 
-A **94-byte** JavaScript enum implementation.
+A **67-byte** JavaScript enum implementation.
+
+Hazer-hazer version of `tolgaek` 94-byte JavaScript enum implementation.
```

Comment:
This downplays the original creators. It'd be kinder to say that @tolgaek and I originally wrote this, and you improved upon it.

Reply:
:)